### DISCUSSION TITLE
Failure processing application bundle - TShock 5 for Terraria 1.4.4.7 (Alnilam) - Windows x64

### DISCUSSION THREAD

**Discussion by Yahm0n** [Pryaxis/TShock#2821] [Q&A]
In TShock 5 for Terraria 1.4.4.7 (Alnilam)
I grabbed the package for my OS: [TShock-5.0.0-for-Terraria-1.4.4.7-win-x64-Release.zip](https://github.com/Pryaxis/TShock/releases/download/v5.0.0/TShock-5.0.0-for-Terraria-1.4.4.7-win-x64-Release.zip)

When I tried to run it, I simply get this error:
```
C:\GameServers\Terraria\1447>TShock.Server.exe
Failure processing application bundle.
Bundle header version compatibility check failed.
A fatal error occured while processing application bundle
```
I'm using Windows 11.  I don't believe I had the Dotnet 6 runtime installed before trying the first time, but the error has not changed since installing it.

**Comment by BeaterLee** [comment]
I have the same problem, I solve it when installed .net runtime 6.0.6(windows desktop runtime)

**Comment by Badagronk** [comment]
I am getting the following error, any advice? Installing 6.0.10 from the link in the error didn't fix it, same error.

C:\Windows\System32>D:\GOG\Terraria\TShock.Server.exe
The required library C:\Program Files\dotnet\host\fxr\3.1.19\hostfxr.dll does not support single-file apps.
  _ To run this application, you need to install a newer version of .NET Core.
  - https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win10-x64&apphost_version=6.0.10
edit: that worked but also needed
https://dotnet.microsoft.com/en-us/download/dotnet/6.0
